### PR TITLE
Do not consider visibility CSS for fixed layer since it's inherited

### LINKED ISSUE #2977: Runtime intermittently fails to create fixed layer leading to jittery scroll on iPhone
I've been seeing intermittent jitter scrolling with a fixed layer on safari/iPhone.

Hooking the phone up to my Mac and debugging I see that on the pages where we see jitter the AMP runtime has failed to move the fixed layer outside the `<body>` tag.  When it does manage to move the layer it scrolls jitter free.

Steps to reproduce:

Load this url on an iPhone and try scrolling.  If Keep reloading and about 20% of the time you'll get a jittery version.

https://www.google.com/amp/s/fresnobee.relaymedia.com/amp/sports/college/mountain-west/fresno-state/article73389102.html?client=safari#scso=uid_0:0

Testing on iPhone 6+ running 9.3.1 (13E238)  using Safari with production (as of 2016-04-22) version of AMP.  `Powered by AMP ⚡ HTML – Version 1460655576651` (both cases)

Screen grab of dom - working version

image

Screen grab - jittery version

image

### PR DESCRIPTION
**Pull request by dvoytenko**
Closes #2977.

### CODE REVIEW

**Comment by erwinmombay** [comment]
@dvoytenko LGTM. out of curiosity, is it AMP that is making it hidden or user land css?

File: test/functional/test-fixed-layer.js [hunk 1]
```diff
@@ -597,6 +597,7 @@ describe('FixedLayer', () => {
       element1.computedStyle['position'] = 'fixed';
       element1.offsetWidth = 10;
       element1.offsetHeight = 10;
+      element1.computedStyle['top'] = '0px';
```
Comment (erwinmombay) [inline test/functional/test-fixed-layer.js]: just 0 ?

**Comment by dvoytenko** [comment]
It's our boilerplate :)

File: test/functional/test-fixed-layer.js (hunk 1 above)
Comment (dvoytenko) [inline test/functional/test-fixed-layer.js]: No, not really - `getComputedStyle` always returns resolved pixel values. So, it's best to mimic it the same way in tests.

**Pull request merged**